templates/projects/stats.html

```diff
@@ -267,17 +267,13 @@ <h4><strong>{{_('Top 10 users')}}</strong></h4>
                         <script>
                         (function(){
                         var container = document.getElementById('userAuthStats');
-                        var graph = Flotr.draw(container, [
-                            {data:[[0, projectStats.userAuthStats['top10'][0]['tasks'] ]],  label:projectStats.userAuthStats['top10'][0]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][1]['tasks'] ]],  label:projectStats.userAuthStats['top10'][1]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][2]['tasks'] ]],  label:projectStats.userAuthStats['top10'][2]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][3]['tasks'] ]],  label:projectStats.userAuthStats['top10'][3]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][4]['tasks'] ]],  label:projectStats.userAuthStats['top10'][4]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][5]['tasks'] ]],  label:projectStats.userAuthStats['top10'][5]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][6]['tasks'] ]],  label:projectStats.userAuthStats['top10'][6]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][7]['tasks'] ]],  label:projectStats.userAuthStats['top10'][7]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][8]['tasks'] ]],  label:projectStats.userAuthStats['top10'][8]['fullname']},
-                            {data:[[0, projectStats.userAuthStats['top10'][9]['tasks'] ]],  label:projectStats.userAuthStats['top10'][9]['fullname']}],
+                        var authUsersGraphData = []
```

Comment:
This could be:
```javascript
var authUsersGraphData = projectStats.userAuthStats.all_users.map(function(user) {
    return { data: [0, user.tasks], label: user.fullname };
});
```